### PR TITLE
Update deploy-scanner-configure-install.md

### PR DESCRIPTION
**Pull request by dbradley-expo**
Updating the working regarding the App secret lifetime

### CODE REVIEW

File: microsoft-365/compliance/deploy-scanner-configure-install.md [hunk 1]
```diff
@@ -232,7 +232,7 @@ For more information, see [How to label files non-interactively for Azure Inform
     > If your scanner service account cannot be granted the **Log on locally** right for the installation, use the *OnBehalfOf* parameter with [Set-AIPAuthentication](/powershell/module/azureinformationprotection/set-aipauthentication), as described in [How to label files non-interactively for Azure Information Protection](/azure/information-protection/rms-client/clientv2-admin-guide-powershell#how-to-label-files-non-interactively-for-azure-information-protection).
     >
 
-The scanner now has a token to authenticate to Azure AD. This token is valid for one year, two years, or never, according to your configuration of the **Web app /API** client secret in Azure AD. When the token expires, you must repeat this procedure.
+The scanner now has a token to authenticate to Azure AD. This token can be valid for a maximum of 2 years, but is set according to your configuration of the **Web app /API** client secret in Azure AD. When the token expires, you must repeat this procedure.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/deploy-scanner-configure-install.md]:
```suggestion
The scanner now has a token to authenticate to Azure AD. This token can be valid for a maximum of two years but is set according to your configuration of the **Web app /API** client secret in Azure AD. When the token expires, you must repeat this procedure.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dbradley-expo, kindly commit the minor suggested changes. Many thanks.

**Review by dbradley-expo — COMMENTED** [review]
Great, thank you

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@libarson Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@libarson Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@dbradley-expo Thanks for your feedback! New issues and pull requests for this content are closed while we update this content to support upcoming feature improvements.

**Pull request closed without merging**